lib/stack_master/parameter_resolvers/ami_finder.rb

```diff
@@ -0,0 +1,34 @@
+module StackMaster
+  module ParameterResolvers
+    class AmiFinder
+      def initialize(region)
+        @region = region
+        @owner = ['self']
+      end
+
+      def build_filters(value, prefix = nil)
+        value.split(',').map do |name_with_value|
+          name, value = name_with_value.strip.split('=')
+          @owner.push name if name == 'owner_id'
```

Comment:
I was expecting `describe_images` to AND the owner and filter, but owner over-rides the filter.